build/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.15.7 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.17.12 as builder
 
 WORKDIR /workspace
 # Copy the Go Modules manifests
 COPY go.mod go.mod
-# COPY go.sum go.sum
+COPY go.sum go.sum
```

Comment:
Why wasn't this needed before ? just for my understanding, please.

Reply:
We moved to go 1.17 I think that's why